lib/vtls/openssl.c

```diff
@@ -1122,7 +1117,14 @@ static int Curl_ossl_init(void)
 {
 #if (OPENSSL_VERSION_NUMBER >= 0x10100000L) &&  \
   !defined(LIBRESSL_VERSION_NUMBER)
-  OPENSSL_init_ssl(OPENSSL_INIT_ENGINE_ALL_BUILTIN | INITFLAGS, NULL);
+  const int64_t flags = OPENSSL_INIT_ENGINE_ALL_BUILTIN |
+#ifdef CURL_DISABLE_OPENSSL_AUTO_LOAD_CONFIG
+    OPENSSL_INIT_NO_LOAD_CONFIG
+#else
+    OPENSSL_INIT_LOAD_CONFIG
+#endif
```

Comment:
BoringSSL didn't approve:

~~~
vtls/openssl.c:1120:25: error: ‘OPENSSL_INIT_ENGINE_ALL_BUILTIN’ undeclared (first use in this function); did you mean ‘OPENSSL_INIT_ADD_ALL_CIPHERS’?
   const int64_t flags = OPENSSL_INIT_ENGINE_ALL_BUILTIN |
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                         OPENSSL_INIT_ADD_ALL_CIPHERS